docs/cvl/statements.md

```diff
@@ -125,6 +125,51 @@ A success does only guarantee that there is some execution starting in some
 arbitrary state.  It is not possible to check that there is an execution for
 every starting state.
 ```
+```{note}
+It is advised to look at the witness example to determine that this is indeed the desired case. 
+```
+For [example](https://gist.github.com/...) 
+https://github.com/Certora/Examples/tree/master/FullProjects 
+the following rule check if there is a scenario in which one can withdraw and get back all his assets
+
+
+```cvl
+rule possibleToFullyWithdraw(address sender, uint256 amount) {
+    env eT0;
+    env eM;
+    uint256 balanceBefore = _token0.balanceOf(sender);
+    
+    require eM.msg.sender == sender;
+    require eT0.msg.sender == sender;
+    _token0.transfer(eT0, currentContract, amount);
+    uint256 amountOut0 = mint(eM,sender);
+    burnSingle(eM, _token0, amountOut0, sender);
+    satisfy balanceBefore == _token0.balanceOf(sender);
+}
+```
```

Comment:
```suggestion
```cvl
//// demonstrate that one can fully withdraw their assets
rule possibleToFullyWithdraw(address sender, uint256 amount) {
    uint256 balanceBefore = token.balanceOf(sender);
    
    env eTransfer;
    require eTransfer.msg.sender == sender;
    token.transfer(eTransfer, currentContract, amount);

    env eMint;
    require eMint.msg.sender == sender;
    uint256 amountOut = mint(eMint,sender);

    burnSingle(eMint, token, amountOut, sender);
    
    satisfy balanceBefore == token.balanceOf(sender);
}
```
```